src/aics-image-viewer/shared/utils/sliderValuesToImageValues.ts

```diff
@@ -32,8 +32,8 @@ export const gammaSliderToImageValues = (sliderValues: [number, number, number])
 // Density and brightness are overloaded for the two supported rendering modes in the volume viewer.
 // These formulae are somewhat ad-hoc based on what will subjectively look reasonable in the viewer, and should be considered tweakable.
 // The volume-viewer expects to see numbers from 0..1 for density, brightness, and alpha.
-export const densitySliderToImageValue = (sliderValues: number[], _isPathTrace: boolean): number => +sliderValues[0] / 100.0;
+export const densitySliderToImageValue = (sliderValues: number, _isPT: boolean): number => +sliderValues / 100.0;
```

Comment:
i guess these were formerly arrays of number because of the nouislider returning its values as array because it can have multiple slider handles